addons/mail/static/src/components/message_action_list/message_action_list.scss

```diff
@@ -0,0 +1,26 @@
+.o_MessageActionList {
+    position: absolute;
+    top: - map-get($spacers, 3);
+    right: map-get($spacers, 2);
+    background-color: $white;
+    border: 1px solid $border-color;
+    border-radius: map-get($spacers, 2);
+    display: flex;
+
+    &:hover {
+        box-shadow: 0 4px .5rem -.5rem black;
+    }
+}
+
+.o_MessageActionList_action {
+    padding: map-get($spacers, 2);
+    cursor: pointer;
+
+    &:hover {
+        background-color: mix($border-color, $white);
+    }
+
+    &.fa-star {
+        color: gold;
+    }
+}
```

Comment:
could review css to use some more classes instead, overall need to make a second validation later